packages/atomic/tailwind.config.js

```diff
@@ -44,15 +44,15 @@ module.exports = {
         '2xl': 'var(--atomic-text-2xl)',
       },
       screens: {
-        'desktop-only': {min: '1023px'},
-        'mobile-only': {max: '1024px'},
+        'desktop-only': {min: '1024px'},
+        'mobile-only': {raw: 'not all and (min-width: 1024px)'},
```

Comment:
Hmm, mobile-only `min-width` is1024px?

Reply:
in media queries, the `not` appears to apply to the entire rest of the query, making this an exclusive `max-width`.